src/bootstrap/bootstrap.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Could these imports go under the license header? Or is it required that they're the first thing in the file?